test/helpers/kubectl.go

```diff
@@ -535,15 +544,20 @@ func (kub *Kubectl) WaitforPods(namespace string, filter string, timeout int64)
 			return false
 		}
 
-		valid := true
+		valid := 0
```

Comment:
Now that you add this one, I would add the valid in the log message in the end, like: 

```golang
kub.logger.WithFields(logrus.Fields{
			"namespace": namespace,
			"filter":    filter,
			"valid":      valid,
"minRequired": minRequired,
}).Info("WaitforPods: pods are not ready")
```